morph_tool/spatial.py

```diff
@@ -22,9 +22,9 @@ def point_to_section_segment(neuron, point):
 
     for section in neuron.iter():
         points = section.points
-        offset = np.where((points[:, X] == point[COLS.X]) &
-                          (points[:, Y] == point[COLS.Y]) &
-                          (points[:, Z] == point[COLS.Z]))
+        offset = np.where(np.allclose(points[:, X], point[COLS.X]) &
```

Comment:
why not
```
np.where(points[:, XYZ] == point[COLS.XYX])
```
?

Reply:
yes, not sure why X, Y, Z was define at the top either, surely benoit's mess again ;)